etc/docker/elyra/Dockerfile

```diff
@@ -23,13 +23,13 @@ ARG TAG="dev"
 ARG ELYRA_VERSION
 ARG ELYRA_PACKAGE=elyra-"$ELYRA_VERSION"-py3-none-any.whl
 
-COPY $ELYRA_PACKAGE requirements.txt ./
+COPY requirements.txt ${ELYRA_PACKAGE}* ./
 
 # Install Elyra
 RUN if [ "$TAG" = "dev" ] ; then \
-    python3 -m pip install --quiet --no-cache-dir "$ELYRA_PACKAGE"[all] && rm $ELYRA_PACKAGE; \
+    python3 -m pip install --quiet --no-cache-dir "${ELYRA_PACKAGE}[all]" && rm ${ELYRA_PACKAGE} ; \
```

Comment:
What does the asterisk do and why is not necessary on the `rm` command following the `pip install` of the wheel file?

Reply:
After some research I found that wildcards enable us to perform a conditional copy. Wildcarding is not required for the removal because the file must exist at that point. If it doesn't exist, `pip install` will fail and `rm` is never invoked.